.travis.yml

```diff
@@ -21,8 +21,8 @@ script:
 after_success:
 - >
   [ "$TRAVIS_BRANCH" = "master" ] &&
-  [ "$TRAVIS_PULL_REQUEST" = "false" ] &&
-    make deploy_doc
+  [ "$TRAVIS_PULL_REQUEST" = "false" ]
```

Comment:
@xiangge Did you accidentally remove the `&&` here?

Reply:
@hluk no, I know I remove it :)